doorway-ui-components/src/locales/general.json

```diff
@@ -687,19 +687,22 @@
   "listings.reservedCommunityTitleDefault": "Reserved Building",
   "listings.reservedCommunityType": "Reserved Community Type",
   "listings.reservedCommunityTypeDescriptions.developmentalDisability": "People with Developmental Disabilities",
+  "listings.reservedCommunityTypeDescriptions.seniorVeterans": "seniors 62+ years old who have served in the US military and who were honorably discharged.",
```

Comment:
I believe this string shows up on the tag on the listings page, by appending the name of the community type - we will likely need something shorter

Reply:
Hey @emilyjablonski I'm not sure if I'm looking in the right place but it looks like: 
![image](https://github.com/metrotranscom/doorway/assets/10088764/826db978-586f-4872-8375-ac5adf9b1162)
 for me locally

Reply:
Oh the message actually doesn't look too bad! Was nervous for mobile but I think it's all good too 👍